hail/src/main/scala/is/hail/types/physical/stypes/SCode.scala

```diff
@@ -112,6 +112,8 @@ trait SValue {
   def st: SType
 
   def get: SCode
+
+  def hash(cb: EmitCodeBuilder): SInt32Code = throw new UnsupportedOperationException(s"Stype ${SType} has no hashcode")
```

Comment:
You have to write `st` instead of `SType` in this error message. Otherwise it'll always print some class information about the `SType` class.